assembl/views/api2/auth.py

```diff
@@ -466,6 +466,11 @@ def assembl_register_user(request):
                 "We already have a user with this username.")),
                 ErrorTypes.EXISTING_USERNAME,
                 HTTPConflict.code)
+        if len(username) > 20:
+            errors.add_error(localizer.translate(_(
+                "The username must be less than 20 characters.")),
+                ErrorTypes.INVALID_USERNAME,
```

Comment:
Hm, add another Enum type to ErrorTypes, `USERNAME_TOO_LONG`. The front-end is normally supposed to parse these and do stuff with them, but it doesn't. Anyways, doesn't hurt to put the proper error type.